openstore/apiconstants.h

```diff
@@ -0,0 +1,15 @@
+#ifndef APICONSTANTS_H
+#define APICONSTANTS_H
+
+#define STORE_DOMAIN QStringLiteral("open.uappexplorer.com")
+#define API_BASEURL QStringLiteral("https://open.uappexplorer.com/api/")
```

Comment:
@sverzegnassi it might be useful to have this picked up from an env var (or something at build time) for hitting a testing server when doing server development.

Reply:
Let's do this with a different PR?

Reply:
Of course, it's mearly me putting toughts on paper so to speak 🙂